packages/fxa-auth-server/lib/routes/recovery-phone.ts

```diff
@@ -147,7 +173,14 @@ class RecoveryPhoneHandler {
    * @param request
    */
   async available(request: AuthRequest) {
-    const { uid } = request.auth.credentials as unknown as { uid: string };
+    const { uid, email } = request.auth
+      .credentials as unknown as SessionTokenAuthCredential;
+
+    if (!email || !uid) {
+      throw AppError.invalidToken();
+    }
+
+    await this.customs.check(request, email, 'recoveryPhoneAvailable');
```

Comment:
@dschom I wonder if we need to do a customs server check on endpoints that are authenticated with sessionToken already. In https://github.com/mozilla/fxa/pull/18171/files#diff-2d2788abe335cf346f57750c4ede3d7375ec5db92ae3fad2bf9d609f50b77e6aL32, we call out unauthenticated requests that check account info.

Reply:
Looking at this a little more, it seems like we need to keep the check but maybe not have it part of the "ACCOUNT_STATUS_ACTION" gropu.